projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -88,21 +95,22 @@
   "item.pamhc2foodextended.grapefruitsodaitem": "西柚汽水",
   "item.pamhc2foodextended.grapesodaitem": "葡萄汽水",
   "item.pamhc2foodextended.hazelnutchocolateitem": "榛子巧克力酱",
-  "item.pamhc2foodextended.lemonlimesodaitem": "柠檬酸橙汽水",
+  "item.pamhc2foodextended.lemonlimesodaitem": "柠檬汽水",
   "item.pamhc2foodextended.maplecandiedbaconitem": "枫糖培根",
   "item.pamhc2foodextended.marzipanitem": "杏仁蛋白软糖",
   "item.pamhc2foodextended.orangesodaitem": "橘子汽水",
   "item.pamhc2foodextended.peanutbuttercupitem": "花生巧克力杯",
   "item.pamhc2foodextended.peppermintitem": "薄荷糖",
   "item.pamhc2foodextended.popcornitem": "爆米花",
-  "item.pamhc2foodextended.pralinesitem": "山核桃糖",
+  "item.pamhc2foodextended.pralinesitem": "碧根果糖",
   "item.pamhc2foodextended.rootbeerfloatitem": "雪顶根汁啤酒",
   "item.pamhc2foodextended.rootbeersodaitem": "根汁汽水",

```

```suggestion
  "item.pamhc2foodextended.rootbeersodaitem": "根汁啤酒",
```
说是`beer`但可能不含酒精。另外还有个译名叫`麦根沙示`，不知道哪个更好一些。

---

刚刚才看见...为什么前一行还是`根汁啤酒`后一行就成了`根汁汽水`了啊（